lib/bom.js

```diff
@@ -261,6 +261,61 @@ declare class Worker extends EventTarget {
     terminate(): void;
 }
 
+declare class SharedWorker extends EventTarget {
+    constructor(stringUrl: string): void;
+    port: MessagePort;
+    onerror: (ev: Event) => any;
+}
+
+declare function importScripts(...urls: Array<string>): void;
+
+declare class WorkerGlobalScope extends EventTarget {
+    self: WorkerGlobalScope;
+    location: WorkerLocation;
+    navigator: WorkerNavigator;
+    importScripts(...urls: Array<string>): void;
+    onerror: (ev: Event) => any;
+    onlanguagechange: (ev: Event) => any;
+    onoffline: (ev: Event) => any;
+    ononline: (ev: Event) => any;
+    onrejectionhandled: (ev: PromiseRejectionEvent) => any;
+    onunhandledrejection: (ev: PromiseRejectionEvent) => any;
+}
+
+declare class DedicatedWorkerGlobalScope extends WorkerGlobalScope {
+    close(): void;
```

Comment:
I was working from the [spec document](https://html.spec.whatwg.org/multipage/workers.html#the-workerglobalscope-common-interface) as a reference and it happened to have `close()` on the subs instead of the shared interface.  I think it would be fine to change though since they're the same signature. Is that something you would like to see changed?